deeposlandia/datasets/tanzania.py

```diff
@@ -242,36 +386,45 @@ def load_mask(self, buildings, raster_features, min_x, min_y):
         if buildings.shape[0] == 0:
             return mask
         for idx, row in buildings.iterrows():
-            points = self.extract_points_from_polygon(row["geometry"],
-                                                      raster_features)
-            points[:, 0] -= min_x
-            points[:, 1] -= min_y
+            points = extract_points_from_polygon(row["geometry"],
+                                                 raster_features,
+                                                 min_x, min_y)
             label_id = [label["id"] for label in self.labels
                         if label["name"] == row["condition"].lower()][0]
             mask = cv2.fillPoly(mask, [points], label_id)
         return mask
 
 
-    def extract_points_from_polygon(self, p, features):
-        """Extract points from a polygon
+def extract_points_from_polygon(p, features, min_x, min_y):
```

Comment:
this function could be tested

Reply:
Hum... That's a big point, as it implies to test geometric coordinates. To be fair, we should even test other functions of the module...
How about a next dedicated PR?

Reply:
In a next dedicated PR it's a good idea. If you create the input data (simple data), the returned data should be tested.

Reply:
See #104